src/command/test.rs

```diff
@@ -183,9 +184,9 @@ impl Test {
         }
         match self.mode {
             BuildMode::Normal => steps![
-                step_check_crate_config,
                 step_add_wasm_target,
                 step_build_tests,
+                step_check_crate_config,
```

Comment:
Thinking about it some more, we might want to still check the crate config as the first thing we do, but have checking crate config *not* check wasm-bindgen versions. Then when we go to install wasm-bindgen, we look at the lockfile at tjhat time.

Reply:
yes i think that's a good call. crate config should def be one of the first things we do because it's a quick fix for folks who it fails for.